react/components/atoms/checkbox/checkbox.js

```diff
@@ -0,0 +1,181 @@
+import React, { PureComponent } from "react";
+import { Platform, StyleSheet, Text, View, ViewPropTypes } from "react-native";
+import PropTypes from "prop-types";
+import { mix } from "yonius";
+
+import { IdentifiableMixin, baseStyles } from "../../../util";
+
+import { Icon } from "../icon";
+import { Touchable } from "../touchable";
+
+export class Checkbox extends mix(PureComponent).with(IdentifiableMixin) {
+    static get propTypes() {
+        return {
+            label: PropTypes.string,
+            checked: PropTypes.bool,
+            disabled: PropTypes.bool,
+            icon: PropTypes.string,
+            size: PropTypes.number,
+            variant: PropTypes.string,
+            onUpdateChecked: PropTypes.func,
+            style: ViewPropTypes.style
+        };
+    }
+
+    static get defaultProps() {
+        return {
+            icon: "check",
+            checked: false,
+            disabled: false,
+            size: 20,
+            variant: undefined,
+            onUpdateChecked: () => {},
+            style: {}
+        };
+    }
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            checkedData: props.checked,
+            active: false
+        };
+    }
+
+    componentDidUpdate(prevProps) {
+        if (prevProps.checked === this.props.checked) return;
+        this.setState({
+            checkedData: this.props.checked
+        });
+    }
+
+    onPressIn = () => {
+        this.setState({ active: true });
+    };
+
+    onPressOut = () => {
+        this.setState(
+            prevState => ({
+                active: false,
+                checkedData: !prevState.checkedData
+            }),
+            () => {
+                this.props.onUpdateChecked(this.state.checkedData);
+            }
+        );
+    };
+
+    _icon = () => {
+        if (this.props.disabled) return `${this.props.icon}-gray`;
+        if (this.state.active) return `${this.props.icon}-blue`;
+        return this.props.icon;
+    };
+
+    _style = () => {

```

Comment:
We're reserving `_style` for the root component, so call this something else (e.g. styleBox)